static/css/style.css

```diff
@@ -35,6 +37,18 @@ footer {
 	text-align: auto;
 }
 
+#address {
+	color: #FFFFFF
+}
+.container-fluid {
+	background-color: #734F43;
+}
+.navbar {
```

Comment:
To make a navigation bar fixed, you can add to the html nav tag add the "navbar-fixed-top" class. and  delete this .navbar definition from the style.css